iroha/Cargo.toml

```diff
@@ -41,6 +41,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 heim = "0.0.11"
 log = { version = "0.4", features = ["std", "serde"] }
+attohttpc = "0.15.0"
```

Comment:
I would suggest to use [isahc](https://docs.rs/isahc/1.0.3/isahc/#asynchronous-requests) as we have asynchronous core and blocking client like attohttpc can introduce some problems by stopping the thread.

Later of course we will replace isahc with our custom http client.